utils/semverutils/semverutils_test.go

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+// Copyright (C) 2024 Renesas Electronics Corporation.
+// Copyright (C) 2024 EPAM Systems, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package semverutils_test
+
+import (
+	"testing"
+
+	"github.com/aosedge/aos_common/utils/semverutils"
+)
+
+/***********************************************************************************************************************
+ * Tests
+ **********************************************************************************************************************/
+
+func TestLessThan(t *testing.T) {
+	res, err := semverutils.LessThan("1.9", "1.10")
```

Comment:
In this case input data should be in array.
No need to add`semverutils.LessThan("1.3", "1.3")` because changing test data requires to change this string as well.

Reply:
fixed input data